ansible/roles/piwik/templates/nginx-site-piwik.j2

```diff
@@ -0,0 +1,67 @@
+server {
+    server_name {{ server_name}};
+
+    listen 443 ssl default_server;
+    listen [::]:443 ssl default_server;
+
+    ssl_certificate /etc/letsencrypt/live/{{ letsencrypt_domains.split(',')[0] }}/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/{{ letsencrypt_domains.split(',')[0] }}/privkey.pem;
+    ssl_trusted_certificate /etc/letsencrypt/live/{{ letsencrypt_domains.split(',')[0] }}/fullchain.pem;
+
+  if ($request_method !~ ^(GET|HEAD|POST)$ ) {
+    return 444;
+  }
+
+  root  {{ app_path }}/current/piwik;
+  index  index.php index.html;
+  # Disallow any usage of piwik assets if referer is non valid.
+  location ~* ^.+\.(?:jpg|png|css|gif|jpeg|js|swf)$ {
+    # Defining the valid referers.
+    #valid_referers none blocked *.mysite.com othersite.com;
+
+    expires max;
```

Comment:
It really looks like bug. It means that `//analytics.ooni.io/piwik.js` NEVER changes and is cached forever.
Does it sound like true statement?